File: src/main/java/cl/ahianf/rankbot/entity/Song.java
/* (C)2022-2023 - Ahian Fernández Puelles*/
package cl.ahianf.rankbot.entity;

import lombok.Data;

@Data
public class Song {
    private Integer id;
    private Integer songId;
    private String title;
    private String album;
    private String artist;
    private Integer artistId;
    private Double elo;
    private String artUrl;
    private boolean enabled;


}

File: rankbot-front/src/app/model/song.ts
export interface Song {
  songA: SongData;
  songB: SongData;
  token: number;
}

export interface SongData {
  id: number;
  songId: number;
  title: string;
  album: string;
  artist: string;
  artistId: number;
  elo: number;
  artUrl: string;
  enabled: boolean;
}
